apps/che/src/main/fabric8/deployment.yml

```diff
@@ -144,13 +144,17 @@ spec:
         imagePullPolicy: "IfNotPresent"
         name: che
         readinessProbe:
-          tcpSocket:
+          httpGet:
+            path: /api/system/state
             port: 8080
-          initialDelaySeconds: 20
+            scheme: HTTP
+          initialDelaySeconds: 60
```

Comment:
@mlabuda I understand your point but that's an optimisation I would look at when we are sure that everything works fine.